libkbfs/kbfs_cr_test.go

```diff
@@ -867,13 +901,15 @@ func TestCRDouble(t *testing.T) {
 	// Cancel this revision after the Put happens, to force the
 	// background block manager to try to clean up.
 	onSyncStalledCh, syncUnstallCh, syncCtx := StallMDOp(
-		syncCtx, config2, StallableMDPutUnmerged, 1)
+		syncCtx, config2, StallableMDAfterPutUnmerged, 1)
 
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
 		err = kbfsOps2.SyncAll(syncCtx, fileNodeC.GetFolderBranch())
-		assert.Equal(t, context.Canceled, err)
+		if err != nil {
+			assert.Equal(t, context.Canceled, err)
```

Comment:
Since `SyncAll` forks a goroutine and then waits on both the goroutine and `ctx.Done()`, there's a race on cancel -- sometimes it will return `nil` if the goroutine finishes first (and ignores the `PutUnmerged` error), and sometimes it will return `context.Canceled`.  Either is ok.